modules/ROOT/pages/constraints/index.adoc

```diff
@@ -1,124 +1,17 @@
-Relationship property type constraints do not require all relationships to have the property (relationships without the property on which the constraint exists are not subject to this rule).
-
-For more information, see xref:constraints/examples.adoc#constraints-examples-relationship-property-type[examples of relationship property type constraints].
-
-[[node-key]]
-[role=label--enterprise-edition]
-== Node key constraints
-
-Node key constraints ensure that, for a given label and set of properties:
-
-. All the properties exist on all the nodes with that label.
-. The combination of the property values is unique.
-
-+
-Queries attempting to do any of the following will fail:
-
-* Create new nodes without all the properties or where the combination of property values is not unique.
-* Remove one of the mandatory properties.
-* Update the properties so that the combination of property values is no longer unique.
-
-For more information, see xref:constraints/examples.adoc#constraints-examples-node-key[examples of node key constraints].
-
-[[relationship-key]]
-[role=label--enterprise-edition label--new-5.7]
-== Relationship key constraints
-
-Relationship key constraints ensure that, for a given type and set of properties:
-
-[lowerroman]
-. All the properties exist on all the relationships with that type.
-. The combination of the property values is unique.
-
-+
-Queries attempting to do any of the following will fail:
-
-* Create new relationships without all the properties or where the combination of property values is not unique.
-* Remove one of the mandatory properties.
-* Update the properties so that the combination of property values is no longer unique.
-
-For more information, see xref:constraints/examples.adoc#constraints-examples-relationship-key[examples of relationship key constraints].
-
-[[multiple-constrains]]
-== Multiple constraints on the same property combinations
-
-
-Some constraint types are allowed on the same label/relationship type and property combination.
-For example, it is possible to have a uniqueness and an existence constraint on the same label/relationship type and property combination, though this would be the equivalent of having a node or relationship key constraint.
-A more useful example would be to combine a property type and an existence constraint to ensure that the property exists and has the given type.
+Neo4j offers several constraints to ensure the quality and integrity of data in a graph.
+The following constraints are available in Neo4j:
 
-[[index-implications]]
-== Implications on indexes
+* *Property uniqueness constraints* ensure that the combined property values are unique for all nodes with a specific label or all relationships with a specific type. 
+* *Property existence constraints* ensure that a property exists either for all nodes with a specific label or for all relationships with a specific type.
+label:enterprise-edition[]
```

Comment:
feels odd to me that the label is on the line below and not the same line as for the other two below